mysql/lib/client.ts

```diff
@@ -162,36 +142,31 @@ export class AwsMySQLClient extends AwsClient {
   }
 
   async setTransactionIsolation(level: TransactionIsolationLevel): Promise<Query | void> {
-    if (level === this.getTransactionIsolation()) {
```

Comment:
Wondering at the extra work involved at setting a value to be the same it currently has?